openaerostruct/docs/installation.rst

```diff
@@ -26,20 +26,51 @@ Then from within the OpenAeroStruct folder, pip install the package:
 
 Both methods will automatically install the dependencies: numpy, scipy, matplotlib, and OpenMDAO.
 
-The latest version of OpenAeroStruct supports OpenMDAO version 3.2.0 to 3.16.0; 3.16.0 is recommended.
-If you wish to install a specific version of OpenMDAO, follow the instructions at https://github.com/OpenMDAO/OpenMDAO/.
+The oldest and latest versions of the dependencies that we test regularly are the following (other versions may work, but no guarantees):
+
+.. list-table::
+    :header-rows: 1
+
+    * - Dependency
+      - oldest
+      - latest
+    * - Python
+      - 3.8
+      - 3.11
+    * - NumPy
+      - 1.20
+      - latest
+    * - SciPy
+      - 1.6.0
+      - latest
+    * - OpenMDAO
+      - 3.10
+      - latest
+    * - Matplotlib
+      - latest
+      - latest
+    * - pyGeo (optional)
+      - 1.6.0
+      - latest
+    * - OpenVSP (optional)
+      - 3.27.1
+      - 3.27.1
```

Comment:
Is this duplicate by mistake?

Reply:
This is intentional since the OpenVSP version is 3.27.1 on both the oldest and latest GitHub Actions builds